### DISCUSSION TITLE
How to have different API URL and Custom endpoint URL

### DISCUSSION THREAD

**Discussion by awnish-npg** [directus/directus#8085] [Archive]
In my application I need to setup the different URL structure say:
API endpoints URL: http://localhost:8056/api/
Custom endpoints URL: http://localhost:8056/custom/ (not http://localhost:8056/api/custom/)
Admin URL: http://localhost:8056/admin/

Please confirm if this is possible or am I overshooting?

**Comment by rijkvanzanten** [comment]
You can route custom endpoints however you like. The admin app is always served on `/admin/` and can't be changed at this point
